packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -92,6 +92,13 @@
               @row-mounted="rowMounted"
               @sort="handleSort"
             >
+              <template #indicators="{ resource }">
```

Comment:
I'd prefer to have the indicators implemented in the `ResourceTable` and `ResourceTiles` components, respectively. I.e. ResourceTable remains unchanged (shows all indicators) while in the ResourceTiles you can do the filtering. The template part is not so much code duplication. In my opinion having the logic (show all indicators vs. show a reduced set of indicators) in the respective components is more important than avoiding the 5-line code duplication.